src/req_rep.rs

```diff
@@ -78,14 +103,21 @@ impl SocketFrontend for ReqSocket {
     fn new() -> Self {
         Self {
             backend: Arc::new(ReqSocketBackend {
-                peers: Arc::new(DashMap::new()),
+                peers: DashMap::new(),
+                round_robin: SegQueue::new(),
+                current_request_peer_id: AtomicCell::new(0),
             }),
             _accept_close_handle: None,
             current_request: None,
         }
     }
 
     async fn bind(&mut self, endpoint: &str) -> ZmqResult<()> {
```

Comment:
Hmm, shouldn't a ZMQ_REQ socket only have `connect()`, and not `bind()`?

Reply:
No all zmq sockets can work both as client or server. It depends on actual use case which variant is used